### DISCUSSION TITLE
各位说3.0版本会不会出爱莉的同位体?

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#4877] [崩坏：星穹铁道]
<img width="360" alt="截屏2024-07-05 21 56 22" src="https://github.com/user-attachments/assets/ef219485-b75c-474f-bcb5-79e541f462ae">

爱莉希雅这个名字正好来源于希腊语，而且翁法罗斯也是来自希腊语

**Comment by March-777th** [comment]
大胆点，2.6联动就出<img src="https://bbs-static.miyoushe.com/static/2024/07/31/2dafa973b0fb29e5342403c4ead8af29_7443113916298582946.png" alt="三月七 比划" width="75" >

**Comment by WuxuRiver** [comment]
这倒是希望，但是爱莉希雅只有一个

**Reply by cyq20091029** [reply]
的确，铁也不可能用一年时间塑造一个人

**Reply by 1234ADS-WQ** [reply]
有啊，我们开拓者小分对<img src="https://img-static.mihoyo.com/communityweb/upload/0b5acb4c4e49bf8733806b11450f6c50.png" alt="三月七_Biu" width="75" >

**Comment by nabeilishi** [comment]
出！一定要狠狠的出啊！！！！！！！！！！！！！！！！！！！！！！！！！！

**Comment by FWJ321** [comment]
真的会有，3.x版本，虽然是内鬼消息，但那个内鬼爆的很准

**Comment by Pardo-Felis-1** [comment]
爱莉姐复活？这个委托我接了！咱绝对不是为了亮闪闪的丁尼（心虚）

**Reply by WuxuRiver** [reply]
> 亮闪闪

谁家小猫
